Revisit page meta descriptions in Core docs - Closes #738 (#753)
LiskHQ/lisk-docs: modules/ROOT/pages/interact-with-the-api.adoc

= Interact with the network
Mona Bärenfänger <mona@lightcurve.io>

> :description: The "Interact with the network" section covers interacting with applications such as Lisk Desktop & Lisk Explorer.

Reviewer: ```suggestion
:description: How to interact with applications such as Lisk Desktop & Lisk Explorer via the Lisk Core API.
```
